decidim-core/lib/decidim/form_builder.rb

```diff
@@ -105,6 +105,139 @@ def categories_select(name, collection, options = {})
 
     private
 
+    # Private: Override from FoundationRailsHelper in order to render
+    # inputs inside the label and to automatically inject validations
+    # from the object.
+    #
+    # attribute    - The String name of the attribute to buidl the field.
+    # options      - A Hash with options to build the field.
+    # html_options - An optional Hash with options to pass to the html element.
+    #
+    # Returns a String
+    def field(attribute, options, html_options = nil, &block)
+      label = options.delete(:label)
+      label_options = options.delete(:label_options)
+      custom_label(attribute, label, label_options) do
+        decidim_field(attribute, options, html_options, &block)
+      end
+    end
+
+    # Private: Builds a form field and detects validations from
+    # the form object.
+    #
+    # attribute    - The String name of the attribute to buidl the field.
+    # options      - A Hash with options to build the field.
+    # html_options - An optional Hash with options to pass to the html element.
+    #
+    # Returns a String.
+    def decidim_field(attribute, options, html_options)
```

Comment:
Wouldn't you use `field_with_validations` in order to make it more explicit? `decidim_field` looks quite generic to me.